chat-plugins/mafia.js

```diff
@@ -1188,6 +1200,33 @@ exports.commands = {
 			`/mafia resethammer - sets the hammer to the default, resetting lynches`,
 		],
 
+		disablenl: 'enablenl',
+		enablenl: function (target, room, user, connection, cmd) {
+			let targetRoom = room;
+			target = target.split(',');
+			if (Rooms(target[0]) && Rooms(target[0]).users[user.userid]) targetRoom = Rooms(target.shift());
+			if (!targetRoom || !targetRoom.game || targetRoom.game.gameid !== 'mafia') return this.errorReply(`There is no game of mafia running in this room.`);
+			if (!user.can('mute', null, room) && targetRoom.game.hostid !== user.userid) return user.sendTo(targetRoom, `|error|/mafia ${cmd} - Access denied.`);
+			if (cmd === 'disablenl') {
+				targetRoom.game.enableNL = false;
+				targetRoom.game.sendRoom(`No-Lynch has been disabled.`, {declare: true});
+				// Remove everyone's lynches from No Lynch
+				if (targetRoom.game.lynches['nolynch']) delete targetRoom.game.lynches['nolynch'];
+				for (const player of Object.values(targetRoom.game.players)) {
+					if (player.lynching === 'nolynch') player.lynching = '';
+				}
+				targetRoom.game.getPlurality();
+				targetRoom.game.updatePlayers();
+			} else {
+				targetRoom.game.enableNL = true;
+				targetRoom.game.sendRoom(`No-Lynch has been enabled.`, {declare: true});
+			}
+		},
+		setnolynchhelp: [
```

Comment:
Not the way to setup a help command try this
```javascript
enablenlhelp: [`/mafia enablenl OR /mafia disablenl - enable or disable players ability to abstain from lynching. Requires host % @ # & ~`],
```

Reply:
mb I was planning to name the cmd setnolynchhelp but I changed it